concepts/basics/introduction.md

```diff
@@ -149,6 +149,8 @@ echo '[{"key":"question", "value":54}, {"key":"answer", "value":42}]' \
 | jq '{(.[0].key): .[0].value, (.[1].key): .[1].value}'
 ```
 
+This also outputs the same as above.
```

Comment:
1. This sentence can be part of the prior paragraph. That would reduce the number of paragraphs/blocks here.
2. This prose should be kept in sync with the `about.md` file.
3. Please ensure you run `./bin/configlet generate` which updates additional files from these.

Reply:
1. Happy to, but is there a style guide that I should read first, before submitting such changes? Reducing the number of paragraphs as an aim wasn't something I would have expected, having seen single word paragraphs elsewhere in this same section. 
2. Sorry, I had no idea that the content was duplicated across files, I just followed the "Edit via GitHub" link from Exercism. Will do.
3. I thought that instruction was for the PR reviewer, not the submitter. I'll do that too.

Reply:
1. There's the various MD docs in the repo root. None are going to say anything about reducing paragraph counts, though. That's not a goal. Rather, related ideas should be bundled into the same paragraph. Paragraphs are used to explain and unify one idea. Having multiple paragraphs to explain one idea and splitting that concept across a code block makes for a less smooth reader experience. This isn't specific to this codebase or project. It's a more generalized writing style idea.
2. The [concepts docs](https://exercism.org/docs/building/tracks/concepts) has details on the file layout.
3. Yes and no? The reviewer could use that to ensure the work was done, but the file change needs to end up in the PR. It's simpler for the author to ensure all changes are part of the PR.

Thanks for your contribution to these docs!